src/components/common/navigation/NavigationRoutes.tsx

```diff
@@ -406,6 +407,9 @@ export default function NavigationRoutes() {
                                             path={URLS.SECURITY}
                                             render={(props) => <Security {...props} serverMode={serverMode} />}
                                         />,
+                                        <Route key={URLS.EVENT_WATCHER} path={URLS.EVENT_WATCHER}>
+                                            {EventWatcherRouter ? <EventWatcherRouter /> : null}
+                                        </Route>,
```

Comment:
This would make the route accessible in non-enterprise. How about (Sorry for the bad formatting):
```suggestion
...(EventWatcherRouter ? [<Route key={URLS.EVENT_WATCHER} path={URLS.EVENT_WATCHER}>
                                             <EventWatcherRouter />
                                        </Route>] : []),
                                        
```

Reply:
Done